hooks/pre/10-reset_data.rb

```diff
@@ -31,14 +31,14 @@ def load_foreman_config
   db_config
 end
 
-def reset_database
-  Kafo::KafoConfigure.logger.info 'Dropping database!'
+def reset_foreman_database
+  logger.info 'Dropping database!'
 
   config = load_foreman_config
   if remote_host?(config[:host])
     empty_database!(config)
   else
-    execute('DISABLE_DATABASE_ENVIRONMENT_CHECK=1 foreman-rake db:drop 2>&1')
+    execute("sudo -u postgres dropdb #{config[:database]}")
   end
```

Comment:
This logic is the same as `reset_pulpcore` and `empty_candlepin_database`, other than the `load_x_config` part. I'd unify that. You can probably also add a more accurate logger there than just `Dropping database!`, which is ambiguous when you're dropping 3 databases.

Reply:
I've unified loading config for all 3 postgresql databases into a single method `load_db_config(db)` and resetting all 3 postgresql databases into a single method `empty_db_in_postgresql(db)`